### PR TITLE
[MIG] sale_coupon_delivery_auto_refresh: Migration to 13.0 

### PR DESCRIPTION
**Pull request by chienandalu**
- [x]  Backport from v14
- [x] Adapt to v13 and fixes

cc @Tecnativa TT37036

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@isserver1 please check

File: sale_coupon_delivery_auto_refresh/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Cetmix
+# Copyright 2022 Ooops404
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Auto Refresh Coupons with Delivery line",
+    "version": "13.0.1.0.0",
+    "summary": "Allows to auto-apply the coupons after adding delivery line",
+    "author": "Ooops404, Cetmix, , Odoo Community Association (OCA)",
+    "contributors": "Ooops404, Cetmix",
+    "license": "LGPL-3",
```
Comment (GabbasovDinar) [inline sale_coupon_delivery_auto_refresh/__manifest__.py]: please, change license to AGPL-3

**Comment by GabbasovDinar** [comment]
LGTM!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge nobump

Something to be changed in the v14 version?

**Pull request merged**